packages/terra-site/CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ Unreleased
 ------------------
 ### Changed
 * Minor version bump
-
+* Updated terra-button-group examples
```

Comment:
We don't need to update the terra-site Changelog. These changes are made in the button-group package